File: python/7 kyu/235.py
# https://www.codewars.com/kata/59ca8e8e1a68b7de740001f4/train/python

# Passed

from collections import Counter

def solve(a,b):
    counter_dict = Counter(a)
    count_arr = []
    for item in b:
        item_count = counter_dict[item]
        count_arr.append(item_count)
        
    return count_arr

output = solve(['abc', 'xyz','abc', 'xyz','cde'], ['abc', 'cde', 'xyz'])
print(output)

File: nodejs/7 kyu/235.js
// https://www.codewars.com/kata/5868812b15f0057e05000001/train/javascript

// Passed

function tailSwap(arr) {
  let [x, y] = arr;
  let xSplit = x.split(":");
  let ySplit = y.split(":");
  let [modifiedX, modifiedY] = [
    xSplit[0] + ":" + ySplit[1],
    ySplit[0] + ":" + xSplit[1],
  ];
  let modifiedArr = [modifiedX, modifiedY];
  return modifiedArr;
}

const output = tailSwap(["abc:123", "cde:456"], ["abc:456", "cde:123"]);
console.log(output);